tests/ops/test_log_linear_attn.py

```
@@ -0,0 +1,53 @@
+):
+    torch.manual_seed(42)
+    os.environ['TRITON_F32_DEFAULT'] = 'ieee'
+
+    x = torch.randn(B, T, H, D, dtype=dtype, device=device)
+    dt = torch.nn.functional.softplus(
+        torch.randn(B, T, H, dtype=torch.float32, device=device) - 4
+    )
+    a = -torch.exp(torch.rand(H, dtype=torch.float32, device=device))
+    q = torch.randn(B, T, 1, D, dtype=dtype, device=device)
+    k = torch.randn(B, T, 1, D, dtype=dtype, device=device)
+    l = torch.randn(B, T, H, L, dtype=dtype, device=device)
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)